collect.py

```diff
@@ -12,8 +12,10 @@
 from selenium.common.exceptions import NoAlertPresentException
 from selenium.common.exceptions import NoSuchWindowException
 from selenium.common.exceptions import TimeoutException
+from selenium.common.exceptions import NoSuchElementException
 
 from autowebcompat import utils
+from lxml import etree
```

Comment:
Moving it in above code segment (where it should be to pass travis) leads to some problem. Only screenshots get captured, dom information is not written, traversing fails. I am not getting how to resolve it. It might be overwriting some previous import but i am not sure how to handle it.

Reply:
Maybe try renaming the import, e.g. `from lxml import etree as lxml_etree`

Reply:
I did try this. It still didn't work. I thought in this way it won't over-write.  :disappointed:

Reply:
Really strange, I'll try to reproduce locally.

Reply:
@marco-c were you able to reproduce the error locally?

Reply:
@marco-c it was lxml version issue. It works fine with 4.2.3 (which was earlier 4.2.2)